Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "CIV-7741_removal_of_paragraph"
```

Comment:
this needs to be reverted before merge to master.

Reply:
Changed it to master